mobileClient/angularMobileApp/src/polyfills.ts

```diff
@@ -0,0 +1,101 @@
+/*
```

Comment:
this file is boilerplate that the angular apps already have. does it need to be duplicated?